lib/parser.rb

```diff
@@ -1,12 +1,31 @@
+require 'json'
+
 class Parser
-  def initialize(file)
-    @file = file
+  def initialize(path)
+    @path = path
   end
 
   def first_line
-    data = File.open(@file)
+    data = File.open(@path)
     content = data.readline
     data.close
     content
   end
+
+  
+  def count_lines_json
+    file = @path.split("/").last
+    lines = count_lines
+    
+    obj = { file => {
+      :lines => lines
```

Comment:
```suggestion
      :lines => count_lines()
```

Se não me engano há uma regra no rubocop pra invocar funções usando o `()` pois poderia ficar ambíguo como se tivesse chamando por uma variável